packages/block/src/header.ts

```diff
@@ -211,6 +211,7 @@ export class BlockHeader {
       }
     }
 
+    // NOTE: The Merge fork block number won't be known prior to the actual fork, since it's determined by total difficulty
```

Comment:
I like the note but not sure if it is relevant in this location

Reply:
Will remove.